assets/src/edit-story/components/helpCenter/useHelpCenter/test/useHelpCenter.js

```diff
@@ -22,138 +22,8 @@ import { renderHook, act } from '@testing-library/react-hooks';
  */
 import APIContext from '../../../../app/api/context';
 import { CurrentUserProvider } from '../../../../app/currentUser';
-import { DONE_TIP_ENTRY, TIPS } from '../../constants';
-import { useHelpCenter, deriveState } from '../';
-
-describe('deriveState', () => {
```

Comment:
Just moved all of this into `test/effects` since derive state is just composed effects